docs/audio.rst

```diff
@@ -215,6 +244,153 @@ Sound Effects Example
 .. include:: ../examples/soundeffects.py
     :code: python
 
+
+AudioRecording & AudioTrack **V2**
+==================================
+
+There are two classes that can contain (or point to) audio data 
+and an associated sampling rate:
+
+- ``AudioRecording`` contains its own buffer, it's initialised with a size
+  defined in time, and it's the object type that ``microphone.record()``
+  returns.
+- ``AudioTrack`` does not hold its own buffer and instead points to a buffer
+  externally created. This buffer could be an ``AudioRecording``, or a basic
+  type like a ``bytearray``. It's similar to a
+  `memoryview <https://docs.micropython.org/en/v1.9.3/pyboard/reference/speed_python.html#arrays>`_
+  and it can be used to easily chop a portion of the audio data in the
+  ``AudioRecording`` or to modify its contents.
+
+AudioRecording
+--------------
+
+.. py:class::
+    AudioRecording(duration, rate=7812)
```

Comment:
Shall we try with a default rate of 11000?

Reply:
Sounds good, will update the docs.